geocode-udf-lambda-aurora/Lambda/geocode.py

```diff
@@ -0,0 +1,82 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+import boto3
+import botocore
+import json
+import logging
+import os
+import time
+
+index_name = os.environ["PLACE_INDEX"]
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+location = boto3.client("location")
```

Comment:
Can you configure this with a custom User Agent so that we're able to identify calls from these UDFs?